mkdocs.yml

```diff
@@ -56,6 +56,11 @@ nav:
   - Contributing: contributing.md
   - Sponsorship: sponsorship.md
 
+extra:
+  analytics:
+    provider: google
+    property: G-KTS6TXPD85
+
```

Comment:
I'm introducing Google Analytics, but happy to replace it if anyone can educate me.